packages/mobile/android/app/src/main/AndroidManifest.xml

```diff
@@ -13,7 +13,7 @@
   <uses-sdk android:minSdkVersion="16" />
 
   <application android:name=".MainApplication" android:allowBackup="false" android:icon="@mipmap/ic_launcher" android:roundIcon="@mipmap/ic_launcher_round" android:label="@string/app_name" android:theme="@style/AppTheme" android:networkSecurityConfig="@xml/network_security_config">
-    <activity android:name=".MainActivity" android:screenOrientation="portrait" android:configChanges="keyboard|keyboardHidden|orientation|screenSize|uiMode" android:label="@string/app_name" android:launchMode="singleInstance" android:theme="@style/AppTheme" android:windowSoftInputMode="adjustResize">
+    <activity android:name=".MainActivity" android:screenOrientation="portrait" android:configChanges="keyboard|keyboardHidden|orientation|screenSize|uiMode" android:label="@string/app_name" android:launchMode="singleTop" android:theme="@style/AppTheme" android:windowSoftInputMode="adjustResize">
```

Comment:
It's to solve this issue: https://github.com/ivpusic/react-native-image-crop-picker/issues/1152
which @isabellewei ran into.

What happens without it is that if you have more than one camera installed (like a custom one and the native one) taking a picture doesn't work. Kind of a weird issue, but this fixes it.

Reply:
Cool! Please add to "Other Changes" in the future.

Reply:
What's the impact of this change for deep links?
> If you wish to receive the intent in an existing instance of MainActivity, you may set the launchMode of MainActivity to singleTask in AndroidManifest.xml. See <activity> documentation for more information.

**https://reactnative.dev/docs/linking#enabling-deep-links**

cc @etuleu

Reply:
Yea, I think react recommends using singleInstance since that guarantees it's basically a singleton. So my worry here would be around making sure we don't have 2 main activities somehow. I am not exactly sure how that could happen, which is also why I don't really see why singleTop would be necessary, unless some other activity goes on top in the same task. In that case maybe singleTask would be good?